testing/scenario_app/android/app/src/androidTest/java/dev/flutter/scenariosui/PlatformTextureUiTests.java

```diff
@@ -99,6 +101,7 @@ public void testPlatformViewRotate() throws Exception {
     intent.putExtra("scenario_name", "platform_view_rotate");
     PlatformViewsActivity activity = activityRule.launchActivity(intent);
     activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+    instrumentation.waitForIdleSync();
```

Comment:
Won't this be null?

I think what you actually want here is `InstrumentationRegistry.getInstrumentation().waitForIdleSync()`

Reply:
(this one is still missing)